examples/view-integration/external-views/src/container/diceRoller/diceRoller.ts

```diff
@@ -69,14 +72,17 @@ export class DiceRollerFactory implements IFluidDataStoreFactory {
 		existing: boolean,
 	): Promise<IFluidDataStoreChannel> {
 		let map: ISharedMap;
+
+		const provideEntryPoint = async (entryPointRuntime: IFluidDataStoreRuntime) => {
+			map ??= (await entryPointRuntime.getChannel(mapId)) as ISharedMap;
```

Comment:
Why do we need `??=` here? Can `provideEntryPoint` be called multiple times?

Reply:
In the "create new" flow, map will get assigned below when it is created for the first time (so we don't need to call getChannel).  It would also be totally OK to just rely on getChannel() again - maybe this would be simpler and worth considering?  I'll take a look.

Reply:
Yeah I think it's not worth the confusion to retain the map reference here, will change in next push.